tests/test_project.py

```diff
@@ -262,9 +262,9 @@ def label2(job):
         assert len(b._label_functions) == 2
         assert len(c._label_functions) == 1
 
+    @pytest.mark.filterwarnings("ignore:always")
     def test_conditions_with_inheritance(self):
         """Tests the inheritance of pre/post conditions.
-
```

Comment:
Add back the empty line you removed for compliance with PEP 257. Multiline docstrings should have a blank line. https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings